samples/tflm_ethosu/src/inference_process.cpp

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (c) 2019-2022 Arm Limited.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "inference_process.hpp"
+
+#include "tensorflow/lite/micro/all_ops_resolver.h"
+#include "tensorflow/lite/micro/cortex_m_generic/debug_log_callback.h"
+#include "tensorflow/lite/micro/micro_error_reporter.h"
+#include "tensorflow/lite/micro/micro_interpreter.h"
+#include "tensorflow/lite/micro/micro_profiler.h"
+#include "tensorflow/lite/schema/schema_generated.h"
+
+#include <cmsis_compiler.h>
+#include <inttypes.h>
+#include <kernel.h>
+
+using namespace std;
+
+namespace
+{
+bool copyOutput(const TfLiteTensor &src, InferenceProcess::DataPtr &dst)
+{
+	if (dst.data == nullptr) {
+		return false;
+	}
+
+	if (src.bytes > dst.size) {
+		printk("Tensor size mismatch (bytes): actual=%d, expected%d.\n", src.bytes,
+		       dst.size);
+		return true;
+	}
+
+	copy(src.data.uint8, src.data.uint8 + src.bytes, static_cast<uint8_t *>(dst.data));
+	dst.size = src.bytes;
+
+	return false;
+}
+
+} // namespace
```

Comment:
As per the [coding style requirements](https://docs.zephyrproject.org/latest/contribute/guidelines.html#coding-style):

> Use C89-style single line comments, /*  */. The C99-style single line comment, //, is not allowed.

Same issue elsewhere in this code.